core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -186,6 +186,9 @@ public int getPosition() {
 
     public void setPosition(int position) {
         this.position = position;
+        if(position > 0 && item.isNew()) {
+            this.item.setPlayed(false);
```

Comment:
So, how does this work with Smart Mark as Played?  Now, any time you've played it at all (regardless of how far) we're going to mark it as played?